_ap/14/index.adoc

```diff
@@ -0,0 +1,14 @@
+---
+num: 14
+category: "SOPs"
+title: "Mitigating exposing sensitive content in upstream SOP repos "
```

Comment:
Agreed. Of course 'sensitive' data should always be prevented from leaking :), but this is more about "enabling the consumers of the SOPs to have more context-specific SOPs for their own purposes. E.g. SREs at a particular company may need a specific task that only makes sense for that company.

Reply:
@emmanuelbernard I believe we are trying to mitigate the potential security threat here that was raised in the ADR document. 

I believe as we have open sourced this SOP process, we also inadvertently opened potential issues such as this security threat of any committer committing private data and therefore we must be very clear about this threat and provide an example on how to mitigate them.

Reply:
Ok yeah that makes sense ill make that change

Reply:
Im not sure i follow the discussion here and maybe i misunderstood the original ask for the AP, but are we not trying to document a easily repeatable model that gives a guide to other Red Hat services on how they could create a upstream repo with steps on how to lower the possibility of exposing Red Hat internal information?

Reply:
>document a easily repeatable model that gives a guide to other Red Hat services on how they could create a upstream repo with steps on how to lower the possibility of exposing Red Hat internal information

@R-Lawton Just to clarify that this is my understanding and I would recommend doing this as it seems it is our responsibility to provide at least an example mitigation model.

Reply:
> So would it be ok for everyone to widen the scope of the AP as I mentioned in the title and explicitly list use cases somewhere and including the sensitive data aspect?

It will list the example mitigation case so it's ok. I agree.

Reply:
The _value_ here isn't so much in "preventing leaks" of internal details (though that _is_ a benefit). Instead it could be very valuable for upstream projects (such as, but certainly not limited to, Strimzi) to maintain SOPs for their software, simply because it's of direct benefit to their users. This allows things like:

* Having these SOPs tested and maintained upstream, where it's least costly
* Letting people in those upstream communities benefit from, contribute to & improve those SOPs, and develop new ones
* Keeping the knowledge of how to effectively operate the software in the community which produces the software.

Where those upstream communities are also upstreams for other Red Hat products then some of this value also flows back to other Red Hat stuff, but I think the case can be made for doing this upstream because it's simply the right thing to do.